.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+contact_links:
+  - name: Usage Support
+    url: https://github.com/semantic-release/semantic-release/discussions
+    about: |
+      For usage and configuration support, please leverage our community discussions.
```

Comment:
i'm wanting to encourage these topics toward discussions. please feel free to suggest improved wording to convey this.
* "how do i...?"
* "my config is this, but it isnt doing what i expect"
* "semantic-release isnt doing what i expect, so i think it is broken (but a maintainer hasnt confirmed whether that is intentional, misconfiguration, or something else)"